templates/saas/model1/src/app/auth/pages/signup.page.tsx

```diff
@@ -1,17 +1,121 @@
+					<path id="ae7af94f-88d7-4204-9f07-e3651de85c05" data-name="Path 99" d="M852.38089,495.2538c-4.28634,2.548-6.85116,7.23043-8.32276,11.9951a113.681,113.681,0,0,0-4.88444,27.15943l-1.55553,27.60021-19.25508,73.1699c16.68871,14.1207,26.31542,10.91153,48.78049-.63879s25.03222,3.85117,25.03222,3.85117l4.49236-62.25839,6.41837-68.03232a30.16418,30.16418,0,0,0-4.86143-4.67415,49.65848,49.65848,0,0,0-42.44229-8.99538Z" transform="translate(-227.576 -76.46149)" fill="#ffffff" />
+					<path d="M846.12661,580.70047a10.52561,10.52561,0,0,1,1.50061.70389l44.34832-22.1972.736-12.02551,18.2938-1.26127.98041,27.4126L852.7199,592.93235a10.4958,10.4958,0,1,1-6.59329-12.23188Z" transform="translate(-227.576 -76.46149)" fill="#ffb8b8" />
+					<path id="a6768b0e-63d0-4b31-8462-9b2e0b00f0fd" data-name="Path 101" d="M902.76552,508.41151c10.91151,3.85117,12.83354,45.57369,12.83354,45.57369-12.8367-7.06036-28.24139,4.49318-28.24139,4.49318s-3.20916-10.91154-7.06034-25.03223a24.52987,24.52987,0,0,1,5.13436-23.10625S891.854,504.558,902.76552,508.41151Z" transform="translate(-227.576 -76.46149)" fill="#ffffff" />
+					<path id="bfd7963f-0cf8-4885-9d3a-2c00bccda2e3" data-name="Path 102" d="M889.99122,467.53052c-3.06-2.44837-7.23517,2.00173-7.23517,2.00173l-2.4484-22.03349s-15.30095,1.8329-25.0935-.61161-11.32255,8.87513-11.32255,8.87513a78.57978,78.57978,0,0,1-.30582-13.77092c.61158-5.50838,8.56838-11.01675,22.6451-14.68932S887.6518,439.543,887.6518,439.543C897.44542,444.43877,893.05121,469.97891,889.99122,467.53052Z" transform="translate(-227.576 -76.46149)" fill="#2f2e41" />
+				</svg> */}
+				<Image src={signup} alt="signup image" className="h-screen"/>
+			</div>
+			<div className="w-max md:w-1/2 py-10 px-5 md:px-10">
+				<div className="text-center mb-10">
+					<h1 className="font-bold text-3xl text-gray-900">Sign Up</h1>
+					<p>Enter your information to sign up</p>
+				</div>
+				<div>
+					<div className="flex flex-wrap -mx-3">
+						<div className="w-full md:w-1/2 px-3 mb-5">
+							<label htmlFor="firstName" className="text-lg font-semibold px-1">First Name</label>
+							<div className="flex">
+								<div className="w-10 z-10 pl-1 text-center pointer-events-none flex items-center justify-center">
+									<i className="text-gray-400 text-lg"><UserIcon/></i>
+								</div>
+								<input id="firstName" type="text" className="w-full -ml-10 pl-10 pr-3 py-2 rounded-lg border-2 border-gray-200 outline-none focus:border-rasengan-primary" placeholder="John" />
+							</div>
+						</div>
```

Comment:
Try to create an Input component which takes:

- Label
- Common Input's props
- Icon

This will help to refactor the code you wrote.

You can base yourself on the templates/dashboard/model1/components/atoms/Input.tsx component